alf/utils/value_ops.py

```diff
@@ -16,6 +16,75 @@
 import tensorflow as tf
 
 from tf_agents.trajectories.time_step import StepType
+from tf_agents.utils import common as tfa_common
+
+from alf.algorithms.rl_algorithm import TrainingInfo
+from alf.utils import common
+
+
+def action_importance_ratio(training_info: TrainingInfo, action_spec,
```

Comment:
for function here, I is better not to using structure like TrainingInfo, because it depends on how particular structure of TrainingInfo. Just pass current_action_distribution and collect_action_distribution directly.

Reply:
Also please make sure the clipped importance ratio is same as the one used by vtrace

Reply:
They are indeed different.  Modified to use different clipping for the different losses.